apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -532,6 +546,49 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
           />
         )}
       />
+      <Controller
+        name="eventTypeColour"
+        render={() => (
+          <SettingsToggle
+            labelClassName="text-sm"
+            toggleSwitchAtTheEnd={true}
+            switchContainerClassName={classNames(
+              "border-subtle rounded-lg border py-6 px-4 sm:px-6",
+              eventTypeColourVisible && "rounded-b-none"
+            )}
+            title={t("event_type_colour")}
+            {...eventTypeColourLocked}
+            description={t("event_type_colour_description")}
+            checked={eventTypeColourVisible}
+            onCheckedChange={(e) => {
+              formMethods.setValue("eventTypeColour", e ? defaultEventTypeColour : null, {
+                shouldDirty: true,
+              });
+              setEventTypeColourVisible(e);
+            }}
+            childrenClassName="lg:ml-0">
+            <div className="border-subtle rounded-b-lg border border-t-0 p-6">
+              <ColorPicker
+                defaultValue={defaultEventTypeColour}
+                onChange={(value) => {
+                  try {
+                    checkWCAGContrastColor("#ffffff", value);
+                    setColourError(false);
+                    formMethods.setValue("eventTypeColour", value, { shouldDirty: true });
+                  } catch (err) {
+                    setColourError(true);
+                  }
+                }}
+              />
+              {colourError ? (
+                <div className="mt-4">
+                  <Alert severity="warning" message={t("light_theme_contrast_error")} />
```

Comment:
> Light Theme color doesn't pass contrast check. We recommend you change this colour so your buttons will be more visible.

Ig we should change this text. We aren't talking about button's visibility here.